common/src/models/response/organizationConnectionResponse.ts

```diff
@@ -0,0 +1,29 @@
+import { OrganizationConnectionType } from "jslib-common/enums/organizationConnectionType";
+
+import { BillingSyncConfigApi } from "../api/billingSyncConfigApi";
+
+import { BaseResponse } from "./baseResponse";
+import { NotificationResponse } from "./notificationResponse";
+
+/**API response config types for OrganizationConnectionResponse */
+export type OrganizationConnectionConfigApis = BillingSyncConfigApi | NotificationResponse;
```

Comment:
What is `NotificationResponse`?

Reply:
oops, thank you, I was testing out the compiler hints with a random response and forgot to delete this 😊